tests/FSharp.Test.Utilities/ScriptingShims.fsx

```diff
@@ -2,13 +2,7 @@ namespace global
 
 [<AutoOpen>]
 module GlobalShims = 
-
-    let errorStringWriter = new System.IO.StringWriter()        
-    let oldConsoleError = System.Console.Error
-    do System.Console.SetError(errorStringWriter)
-
     let exit (code:int) = 
-        System.Console.SetError(oldConsoleError)
-        if code=0 then 
+        if code = 0 then 
             () 
-        else failwith $"Script called function 'exit' with code={code} and collected in stderr: {errorStringWriter.ToString()}"
```

Comment:
Large outputs can be now just printed to the test-specific stdout instead of being passed as exception messages.

Reply:
This is actually really cool. Do you think this can be also separated? Or are you preparing more assorted test improvements? :)

Reply:
This functionality is unfortunately dependent on the code under XUNIT_EXTRAS conditional define. xUnit v3 is expected to have this functionality by default, but in v2 it needs some extending.

The situation with this PR regarding console handling is like this:
Capturing and inspecting outputs in parallel (or sequentially) does work without XUNIT_EXTRAS, so the whole test suite works fine with or without.
`printf` directly to test output (instead of using xUnit's `ITestOutputHelper`) requires the extension of XUNIT_EXTRAS.